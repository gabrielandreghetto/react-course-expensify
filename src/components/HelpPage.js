import React from 'react';

const HelpPage = () => (
    <div>
        this is my help

    </div>
)

export default HelpPage;