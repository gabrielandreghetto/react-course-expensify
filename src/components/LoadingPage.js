import React from 'React';

const LoadingPage = () => (
    <div className="loader" >
        <img src="/images/loader.gif" className="loader__image" />
    </div>
);

export default LoadingPage;