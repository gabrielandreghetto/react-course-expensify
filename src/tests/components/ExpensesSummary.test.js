import React from 'react';
import { shallow } from 'enzyme';

import {ExpensesSummary} from '../../components/ExpensesSummary';
import expenses from '../fixtures/expenses';

test('should correctly render with one expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={23556} />);
    expect(wrapper).toMatchSnapshot();

});

test('should correctly render with multiple expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={12} expensesTotal={25453556} />);
    expect(wrapper).toMatchSnapshot();
});