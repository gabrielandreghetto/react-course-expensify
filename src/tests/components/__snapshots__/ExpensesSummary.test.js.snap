// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render with multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    12
     
    expenses
     totalling 
    $ 254,535.56
  </h1>
</div>
`;

exports[`should correctly render with one expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $ 235.56
  </h1>
</div>
`;
